src/style.css

```diff
@@ -0,0 +1,31 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.button {
```

Comment:
You could write button inside card logic, like class="card__link" or class="card__buy" etc...